recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+
+import os
+from conans import CMake, tools
+from conans.model.conan_file import ConanFile
+
+
+class Embree(ConanFile):
+    name = "embree"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake"
+    homepage = "https://embree.github.io/"
+    _source_folder = "source_subfolder"
+    _build_folder = "build_subfolder"
+    exports_sources = "CMakeLists.txt"
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "tbb/2019_u9"
+
+    options = {
+        "shared": [
+            True,
+            False
+        ],
+        "fPIC": [
+            True,
+            False
+        ],
+        "backface_culling": [
+            True,
+            False
+        ],
+        "ignore_invalid_rays": [
+            True,
+            False
+        ],
+        "ray_masking": [
+            True,
+            False
+        ]
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "backface_culling": False,
+        "ignore_invalid_rays": False,
+        "ray_masking": False
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        target_name = "{0}-{1}".format(self.name, self.version)
+        os.rename(target_name, self._source_folder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+
+        # Configure CMake library build:
+        cmake.definitions["EMBREE_STATIC_LIB"] = not self.options.shared
+        cmake.definitions["BUILD_TESTING"] = False
+        cmake.definitions["EMBREE_TUTORIALS"] = False
+        cmake.definitions["EMBREE_GEOMETRY_CURVE"] = True
+        cmake.definitions["EMBREE_GEOMETRY_GRID"] = True
+        cmake.definitions["EMBREE_GEOMETRY_INSTANCE"] = True
+        cmake.definitions["EMBREE_GEOMETRY_QUAD"] = True
+        cmake.definitions["EMBREE_GEOMETRY_SUBDIVISION"] = True
+        cmake.definitions["EMBREE_GEOMETRY_TRIANGLE"] = True
+        cmake.definitions["EMBREE_GEOMETRY_USER"] = True
+        cmake.definitions["EMBREE_RAY_PACKETS"] = True
+        cmake.definitions["EMBREE_RAY_MASK"] = self.options.ray_masking
+        cmake.definitions["EMBREE_BACKFACE_CULLING"] = self.options.backface_culling
+        cmake.definitions["EMBREE_IGNORE_INVALID_RAYS"] = self.options.ignore_invalid_rays
+        cmake.definitions["EMBREE_ISPC_SUPPORT"] = False
+
+        cmake.configure(build_folder=self._build_folder)
+        return cmake
+
+    def build(self):
```

Comment:
```suggestion
    def build(self):
        os.remove(os.path.join(self._source_folder, "common", "cmake", "FindTBB.cmake"))
```
use FindTBB.cmake provided by Conan

Reply:
added